v0/destinations/active_campaign/transform.js

```diff
@@ -131,6 +131,48 @@ const customTagProcessor = async (message, category, destination) => {
     res.data.tags.map(t => {
       storedTags[t.tag] = t.id;
     });
+    const promises = [];
```

Comment:
Please add inline documentation about this

Reply:
Added